src/solvers/dgmulti/dg_parabolic.jl

```diff
@@ -311,7 +324,9 @@ function calc_divergence_volume_integral!(du, u, flux_viscous, mesh::DGMultiMesh
       # rotate flux to reference coordinates
       fill!(local_viscous_flux, zero(eltype(local_viscous_flux)))
       for j in eachdim(mesh)
-        @. local_viscous_flux = local_viscous_flux + dxidxhatj[j, i][:, e] * flux_viscous[j][:, e]
+        for node in eachindex(local_viscous_flux)
+          local_viscous_flux[node] = local_viscous_flux[node] + dxidxhatj[j, i][node, e] * flux_viscous[j][node, e]
+        end
```

Comment:
Maybe views and broadcasting if you prefer them?

Reply:
I think I'm ok with the loop for now. I was just a bit annoyed that slicing and broadcasting allocated - it seems like it does that for some array types, but not all.